assemblies/assembly-ba-artificial-intelligence.adoc

```diff
@@ -0,0 +1,57 @@
+[id='assembly-ba-artificial-intelligence']
+= Integrating artificial intelligence with {PRODUCT}
+ifdef::context[:parent-context: {context}]
+
+:context: artificial-intelligence
+
+= What is artificial intelligence?
+When you think about artificial intelligence (AI), machine learning and big data might come to mind. But machine learning is only part of the picture. Artificial intelligence encompasses the following technologies:
+* Robotics: The integration of technology, science, and engineering that produces machines that can perform physical tasks that are performed by humans
+* Machine learning: The ability of a collection of algorithms to learn or improve when exposed to data without being explicitly programmed to do so
+* Natural language processing: A subset of machine learning that processes human speech
+* Mathematical optimization: The use of conditions and constraints to find the optimal solution to a problem
+* Digital decisioning: The use of defined criteria, conditions, and a series of machine and human tasks to make decisions
+
+While science fiction is filled with what is referred to as pure AI, machines that perform better than people and cannot be distinguished from them and learn and evolve without human intervention or control, pure AI is decades away. Meanwhile, we have pragmatic AI which is a lot less frightening and much more useful to us today. Pragmatic AI is a collection of AI technologies that combined provide solutions to problems such as predicting customer behavior, providing automated customer service, and helping customers make purchasing decisions.
+
+Leading industry analysts report that previously organizations have struggled with AI technologies because they invested in the potential of AI rather than the reality of what AI can deliver today. AI projects were not productive and as a result investment in AI slowed and budgets for AI projects were reduced. Organizations are now seeing the practical reality of what AI can deliver. Being pragmatic means being practical and realistic.  A pragmatic approach to AI considers AI technologies that are available today, combines them where useful, and adds human intervention when needed to create solutions to real-world problems.
```

Comment:
you might want to make a reference here to the term "AI winter" for the disillusionment phase(s) described